### DISCUSSION TITLE
Pimcore Save settings issue

### DISCUSSION THREAD

**Discussion by remyahalza** [pimcore/pimcore#11352] [Q&A / Support]
I found one issue in pimcore
 
![image](https://user-images.githubusercontent.com/68893691/152287118-a8d37642-fcd3-4e36-a89d-0eaf956f01ca.png)

 
When trying to save settings it shows the above error. But it doesn’t show any other logs.
 
So I can’t able to enable the other languages.

**Comment by dvesh3** [comment]
please check logs under `var/log` to debug the issue. Not reproducible on [demo.pimcore.fun](https://demo.pimcore.fun/admin)
